src/Module/Setup.php

```diff
@@ -0,0 +1,326 @@
+<?php
+/**
+ * @file src/Core/System.php
+ */
+namespace Friendica\Module;
+
+use Friendica\BaseObject;
+use Friendica\Core\System;
+use Friendica\Core\L10n;
+use Friendica\Database\DBStructure;
+use Friendica\Object\Image;
+use Friendica\Util\Network;
+
+
+/**
+ * @file include/Module/Setup.php
+ *
+ * @brief Contains the class with setup relevant stuff
+ */
+
+
+/**
+ * @brief Setup methods
+ */
+class Setup extends BaseObject

```

Comment:
May I ask what your reasoning to call this class `Setup` was? As it stands, we already are using the term `install` for setting up Friendica, for better or worse.

Reply:
If they have the same purpose, it makes sense to me to call them the same. The type of the file/class should help distinguish both.